app.py

```diff
@@ -109,79 +123,78 @@ def update_graph(value, start_date, end_date):
     # else :
     #     # dff = dff[dff['home team'].isin([]])]
 
-    if value is None:
+    if teams is None:
```

Comment:
This if statement and the elif statement below can be combined into one. 

`if teams is None or len(teams) < 2:`